loadtest/control/simulcontroller/actions.go

```diff
@@ -1813,3 +1813,24 @@ func (c *SimulController) getInsights(u user.User) control.UserActionResponse {
 
 	return control.UserActionResponse{Info: fmt.Sprintf("viewed insights for user id %s", userID)}
 }
+
+func (c *SimulController) createPostReminder(u user.User) control.UserActionResponse {
+	ch, err := u.Store().CurrentChannel()
+	if err != nil {
+		return control.UserActionResponse{Err: control.NewUserError(err)}
+	}
+
+	post, err := u.Store().RandomPostForChannel(ch.Id)
+	if err != nil {
+		return control.UserActionResponse{Err: control.NewUserError(err)}
+	}
```

Comment:
Non-blocker: 
Should this be a user action error if no posts are found in the channel store? 
IIUC, user action error counts towards to the error rate that decides when a load test should stop, so maybe this should be no-op instead of returning an error.

Reply:
Yeah good point, I can check that the channel does have posts. Thanks.